src/app/calculator/furnaces/heat-cascading/heat-cascading-form/heat-cascading-form.component.html

```diff
@@ -192,26 +192,26 @@
       </span>
     </div>
 
-    <!-- <div class="form-group">
```

Comment:
I can't find anything explaining why it was commented out. Need some communication about if this was confirmed with Kristina, or is no longer needed.